tools/packaging/common/istio-start.sh

```diff
@@ -77,7 +77,7 @@ if [ "${ISTIO_CUSTOM_IP_TABLES}" != "true" ] ; then
     if [[ ${1-} != "run" ]] ; then
       # clean the previous Istio iptables chains. This part is different from the init image mode,
       # where the init container runs in a fresh environment and there cannot be previous Istio chains
-      "${ISTIO_BIN_BASE}/pilot-agent" istio-clean-iptables
+      "${ISTIO_BIN_BASE}/pilot-agent" istio-iptables --cleanup-only
```

Comment:
This could have been simplified in one  ""${ISTIO_BIN_BASE}/pilot-agent" istio-iptables --reconcile" cmd but I'm keeping the preemptive cleanup+apply step separate to be as consistent as possible with the old handling